Tests/swift-sdk-swift-tests/IterableAutoRegistrationTests.swift

```diff
@@ -67,21 +60,18 @@ class IterableAutoRegistrationTests: XCTestCase {
     }
     
     func testDoNotCallDisableAndEnableWhenSameValue() {
-        let expectation = XCTestExpectation(description: "testDoNotCallDisableAndEnableWhenSameValue")
+        let expectation1 = expectation(description: "registerForRemoteNotifications")
+        expectation1.expectedFulfillmentCount = 1
```

Comment:
1 is the default, do we need this?

Reply:
yes explicit setting is not needed here.